From 83ada4af2a93242610fb99cc64e590d7e44514e1 Mon Sep 17 00:00:00 2001
From: Maurizio Lombardi <mlombard@redhat.com>
Date: Mon, 8 Sep 2014 11:07:37 -0400
Subject: [scsi] bnx2fc: Updated version to 2.4.2

Message-id: <1410174472-14981-3-git-send-email-mlombard@redhat.com>
Patchwork-id: 91635
O-Subject: [RHEL7.1 e-stor PATCH 02/17] [SCSI] bnx2fc: Updated version to 2.4.2
Bugzilla: 1089399
RH-Acked-by: Ewan Milne <emilne@redhat.com>
RH-Acked-by: Rob Evers <revers@redhat.com>

From: Eddie Wai <eddie.wai@broadcom.com>

Old version: 2.4.1
New version: 2.4.2

Signed-off-by: Eddie Wai <eddie.wai@broadcom.com>
Signed-off-by: James Bottomley <JBottomley@Parallels.com>
(cherry picked from commit 38e83bff1439ded2ada7cae166ef5513a6d82226)
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/scsi/bnx2fc/bnx2fc.h b/drivers/scsi/bnx2fc/bnx2fc.h
index 2e984e3..6a97665 100644
--- a/drivers/scsi/bnx2fc/bnx2fc.h
+++ b/drivers/scsi/bnx2fc/bnx2fc.h
@@ -64,7 +64,7 @@
 #include "bnx2fc_constants.h"
 
 #define BNX2FC_NAME  "bnx2fc"
-#define BNX2FC_VERSION  "2.4.1"
+#define BNX2FC_VERSION  "2.4.2"
 
 #define PFX   "bnx2fc: "
 
diff --git a/drivers/scsi/bnx2fc/bnx2fc_fcoe.c b/drivers/scsi/bnx2fc/bnx2fc_fcoe.c
index 0c03f5e..5691414 100644
--- a/drivers/scsi/bnx2fc/bnx2fc_fcoe.c
+++ b/drivers/scsi/bnx2fc/bnx2fc_fcoe.c
@@ -22,7 +22,7 @@ DEFINE_PER_CPU(struct bnx2fc_percpu_s, bnx2fc_percpu);
 
 #define DRV_MODULE_NAME  "bnx2fc"
 #define DRV_MODULE_VERSION BNX2FC_VERSION
-#define DRV_MODULE_RELDATE "Sep 17, 2013"
+#define DRV_MODULE_RELDATE "Dec 11, 2013"
 
 
 static char version[] =
-- 
1.7.1